dll/win32/msgina/lang/ro-RO.rc

```diff
@@ -16,7 +16,7 @@ FONT 8, "MS Shell Dlg", 400, 0, 1
 BEGIN
     CONTROL IDI_ROSLOGO, IDC_ROSLOGO, "Static", SS_BITMAP, 0, 0, 275, 54
     CONTROL "", IDC_BAR, "Static", SS_OWNERDRAW, 0, 44, 275, 4
-    LTEXT "", IDC_STATUS_MESSAGE, 7, 59, 261, 12, SS_WORDELLIPSIS
+    LTEXT "", IDC_LOCKED_MESSAGE, 7, 59, 261, 12, SS_WORDELLIPSIS
```

Comment:
Why did you change IDC_STATUS_MESSAGE to IDC_LOCKED_MESSAGE here?

Reply:
Sorry, I didn't pay attention. I mistakenly did it.